dialogue-core/src/main/java/com/palantir/dialogue/core/InstrumentedChannel.java

```diff
@@ -66,18 +60,11 @@ public void onFailure(Throwable _throwable) {
 
                     private void record(Endpoint endpoint) {
                         long micros = stopwatch.elapsed(TimeUnit.MICROSECONDS);
-                        registry.timer(name(endpoint.serviceName())).update(micros, TimeUnit.MICROSECONDS);
+                        metrics.response(endpoint.serviceName()).update(micros, TimeUnit.MICROSECONDS);
```

Comment:
rambling: I like the way conjure-undertow allows us to build this kind of data into the Endpoint, I wonder if there's a clean way we could do the same thing in dialogue.